desktop/release-notes.md

```diff
@@ -24,12 +24,53 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.11.1
+2022-08-05
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed `docker login` to private registries when Registry Access Management is enabled and access to Docker Hub is blocked [docker/for-win#12871](https://github.com/docker/for-win/issues/12871)
+
```

Comment:
There are two critical bug fixes in this release, why is only one listed?

Reply:
Sorry, my bad, I must have missed the release note field in my PR.

`- Fixed regression preventing VM system locations (e.g. /var/lib/docker) from being bind mounted [for-mac/issues#6433](https://github.com/docker/for-mac/issues/6433)`